src/jetson/requirements.txt

```diff
@@ -0,0 +1,11 @@
+#This file is the required python dependencies installed by docker
+
+pycuda
+pillow
+tqdm==4.42.1
+pycocotools
+pycrypto
+pbkdf2
+distro
+salt
+numpy
```

Comment:
evreything in here needs a version you can get them by running `pip freeze` . after doing this validate that your code still works